libs/net/dal/Migrations/20231004001028_1.0.71.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using TNO.DAL;
+
+#nullable disable
+
+namespace TNO.DAL.Migrations
+{
+    /// <inheritdoc />
+    public partial class _1071 : SeedMigration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            PreUp(migrationBuilder);
+            migrationBuilder.AddColumn<long>(
+                name: "content_id",
```

Comment:
Added foreign key so that we can execute join and where statements related to content.